file-storage.js

```diff
@@ -0,0 +1,16 @@
+// The Following code t
+const contactForm = document.querySelector('form')
+const fnameVal = document.getElementById('f_name')
+const lnameVal = document.getElementById('l_name')
+const emailVal = document.getElementById('mail')
+const msgVal = document.getElementById('msg')
+
+contactForm.addEventListener('input', () => {
+  const formData = {
+    firstName: fnameVal.value,
+    lastName: lnameVal.value,
+    email: emailVal.value,
+    message: msgVal.value
+  }
+  localStorage.setItem('userData', JSON.stringify(formData))
+})
```

Comment:
- [ ] [REQUIRED] Kindly note that you are only setting the data/item to localStorage at the moment. A part of the project requirement also states that when the user loads the page, the code checks if there are any saved data in the local storage, and if so the input fields are pre-filled with the data. Kindly implement this with the window.onload() method. Below is a typical example:
```
window.onload = () => {
  let savedFormData = localStorage.getItem('contactFormInput');
  savedFormData = JSON.parse(savedFormData);
  // Check if the form data object is found on localStorage
  if (savedFormData) {
  // populate inputs values if data was found
  // ex: nameInput.value = savedFormData.name
  }
};
```

Reply:
Ok, thanks for the suggestion. I'll definitely address that.